constants/Styles.ts

```diff
@@ -6,11 +6,23 @@ const styles = StyleSheet.create( {
         flex: 1,
         alignItems: 'center',
         justifyContent: 'center',
-        },
+    },
     title: {
-        fontSize: 20,
+        fontSize: 32,
+        // Not a system font on Android
+        // fontFamily: 'Gill Sans MT',
+        marginHorizontal: 25,
+        marginVertical: '5%',
```

Comment:
I do not think it is good practice to use relative margin.

Reply:
Why, I think this is fine as long as it is responsive to different screen sizes

Reply:
Okay, we'll keep it then.